source/2019-12-30-countdown-to-the-new-year-2019-ember-exam.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Countdown to The New Year: ember-exam
+author: Jessica Jordan
+tags: 2019, Addons, DecEmber, Testing
+alias: 2019/12/30/countdown-to-the-new-year-2019-ember-exam
+responsive: true
+---
+
+This is already the second to last edition of our December series - "Countdown to The New Year: 31 Days of Ember Addons".  In this series we are highlighting a new addon each day until the new year, and we hope you'll join us for the fun!
```

Comment:
```suggestion
This is already the second to last edition of our December series - "Countdown to The New Year: 31 Days of Ember Addons". In this series we are highlighting a new addon each day until the new year, and we hope you'll join us for the fun!
```

^ extra space there

Look good! In more of a backend-focused test suite now, but I love ember-exam. I really like the EmberMap video on the [README](https://github.com/ember-cli/ember-exam) if you want to mention it, but feel free to ignore me as well

Reply:
That sounds like a great addition as well - thank you for suggesting it! I'll update this accordingly